requirements.txt

```diff
@@ -5,7 +5,7 @@ PyYAML
 requests
 six >= 1.7
 kazoo==2.2.1
-python-etcd==0.4.3
+python-etcd==0.4.5
```

Comment:
@CyberDem0n thanks for the clarifications. I think that having `etcd>=0.4.3` is fine. I've updated the pull request.